content/guides/mobile/ios/quick-start/4-make-api-call.md

```diff
@@ -0,0 +1,127 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Make an API call
+
+With our Box and iOS applications created and configured with the **Box iOS
+SDK** we can now make our first call to Box APIs.
+
+Using our blank iOS application, we will create a button to trigger
+a request to Box to fetch the name of the currently authenticated user.
+
+<Message warning>
+  As you integrate Box API calls into your iOS app, this blocking method
+  should be replaced with proper task delegation and non-blocking actions.
```

Comment:
Not sure this read right. Maybe rephrase to actually talk about the implication? "Using a blocking actions as we use in these examples is slow. In a production app these blocking actions would need to be replaced with..."